src/authtoken/views.py

```diff
@@ -0,0 +1,36 @@
+import jwt
+
+from django.conf import settings
+from django.utils import timezone
+from rest_framework import parsers, renderers
+from rest_framework.response import Response
+from rest_framework.views import APIView
+
+from .serializers import CardTokenSerializer
+
+
+class ObtainFooCardToken(APIView):
+    parser_classes = (
+        parsers.FormParser,
+        parsers.MultiPartParser,
+        parsers.JSONParser
+    )
+    renderer_classes = (renderers.JSONRenderer,)
```

Comment:
Any reason for overriding `parser_classes` and `render_classes`? Aren't the DRF defaults good enough?